package.json

```diff
@@ -76,7 +76,7 @@
     "axios": "^0.21.1",
     "classnames": "^2.2.6",
     "compass-mixins": "^0.12.10",
-    "dinero.js": "^1.7.0",
+    "dinero.js": "2.0.0-alpha.8",
```

Comment:
i think we probably don't want to merge without a stable version here

Reply:
Definitely! This will be upgraded to a stable version before this gets promoted to a proper PR that's ready for review. Check out https://github.com/jeremyckahn/farmhand/issues/171#issuecomment-896038011 for the full context for this PR!